core/ledger/kvledger/txmgmt/privacyenabledstate/snapshot_test.go

```diff
@@ -643,3 +645,98 @@ func TestSnapshotImportErrorPropagation(t *testing.T) {
 		require.Contains(t, err.Error(), "error writing batch to leveldb")
 	})
 }
+
+//go:generate counterfeiter -o mock/snapshot_pvtdatahashes_consumer.go -fake-name SnapshotPvtdataHashesConsumer . snapshotPvtdataHashesConsumer
+type snapshotPvtdataHashesConsumer interface {
+	SnapshotPvtdataHashesConsumer
+}
+
+func TestSnapshotImportPvtdataHashesConsumer(t *testing.T) {
+	var dbEnv *LevelDBTestEnv
+	var snapshotDir string
+	var cleanup func()
+	var err error
+
+	init := func() {
+		dbEnv = &LevelDBTestEnv{}
```

Comment:
may not cover  `versionFromSnapshotValue = statecouchdb.VersionFromSnapshotValue`?

Reply:
What I meant was different. If we remove `versionFromSnapshotValue = statecouchdb.VersionFromSnapshotValue` this line from db.go by mistake in the future, all tests in this package will pass. I know that we would not simply remove it but keeping that under check is good IMO.

Reply:
Ah, sure. yes, makes sense covering this scenario